<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>715</width>
    <height>676</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="VolUpButton">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>200</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>VolUp</string>
    </property>
   </widget>
   <widget class="QPushButton" name="VolDownButton">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>260</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>VolDown</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ListenButton">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>320</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>listen</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ChangeLanButton">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>380</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>change language</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>70</y>
      <width>321</width>
      <height>551</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../pic.qrc">:/pic/picture/muschel.png</pixmap>
    </property>
   </widget>
   <widget class="QSlider" name="CordSlider">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>440</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="../pic.qrc"/>
 </resources>
 <connections/>
</ui>
